WordPress/Classes/ViewRelated/Comments/CommentsViewController.h

```diff
@@ -1,14 +1,10 @@
 #import <Foundation/Foundation.h>
-
-#import "CommentsTableViewDelegate.h"
-#import "WPTableViewController.h"
 #import "Blog.h"
```

Comment:
Maybe move the import into the implementation and define it here via a `@class` ?